packager/src/main/java/io/cdap/hub/Tool.java

```diff
@@ -19,6 +19,9 @@
 import com.google.common.base.Splitter;
 import com.google.common.io.BaseEncoding;
 import com.google.common.primitives.Longs;
+import io.cdap.hub.publisher.Publisher;
+import io.cdap.hub.publisher.PublisherFactory;
+import io.cdap.hub.publisher.S3Publisher;
```

Comment:
Since this is abstracted, I feel like that import is not needed?

Reply:
Oh I see that `Tool: getPublisher` returns that type. Should it be changed?

Reply:
yes forgot to remove that unused method, good spot